indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -214,6 +218,9 @@ public boolean run()
 
   public static class IndexGeneratorMapper extends HadoopDruidIndexerMapper<BytesWritable, Text>
   {
+    final QueryGranularity rollupGranularity = getConfig().getGranularitySpec().getQueryGranularity();
```

Comment:
Making the thing a method should work (getConfig().getGranularitySpec().getQueryGranularity() probably isn't that expensive compared to all the other crazy stuff we're doing…)

Or if you want to cache it, lazy initializing